src/libostree/ostree-fetcher-curl.c

```diff
@@ -912,14 +939,10 @@ initiate_next_curl_request (FetcherRequest *req, GTask *task)
   g_assert_cmpint (rc, ==, CURLM_OK);
   rc = curl_easy_setopt (req->easy, CURLOPT_CONNECTTIMEOUT, 30L);
   g_assert_cmpint (rc, ==, CURLM_OK);
-  /* We used to set CURLOPT_LOW_SPEED_LIMIT and CURLOPT_LOW_SPEED_TIME
-   * here, but see https://github.com/ostreedev/ostree/issues/878#issuecomment-347228854
-   * basically those options don't play well with HTTP2 at the moment
-   * where we can have lots of outstanding requests.  Further,
```

Comment:
The core question now is though...is that bug still present?  I am dimly remembering now that this just broke when we ended up doing hundreds of concurrent requests for objects.

But that itself may have already been fixed by 97e9bc7fd4a73c5c63e5390b3935cf986432ab6e !

Reply:
82e2150b98bd1ad5149c1e86db22f8ffb2dd6a41 came after c18628ecb8b2ef591db0444fa57052cba60807a8 so I guess that theory that the low speed limit bug is mitigated is probably wrong.

However...I wonder if it'd be helpful to drop our HTTP queuing from 8 to 2/3.  In practice I doubt that would slow things down much, and it'd avoid having curl thinking that the remote server is answering too slowly - or we've hit a hard cap on the number of concurrent streams.

For reference it looks like the latest Apache defaults too 100 concurrent streams: https://httpd.apache.org/docs/2.4/mod/mod_http2.html
which is quite large.  But I could imagine other servers have this much smaller.

Reply:
Yeah let's definitely make the max concurrent configurable too, would make sense to do on top of this PR